### PR TITLE
Added Dataproc serverless based acceptance tests

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: spark-bigquery-parent/pom.xml [hunk 1]
```diff
@@ -56,8 +56,8 @@
         <gax.version>2.12.0</gax.version>
         <google-cloud-bigquery.version>2.9.4</google-cloud-bigquery.version>
         <google-cloud-bigquerystorage.version>2.11.0</google-cloud-bigquerystorage.version>
-        <google-cloud-dataproc.version>2.3.1</google-cloud-dataproc.version>
-        <google-cloud-storage.version>2.3.0</google-cloud-storage.version>
+        <google-cloud-dataproc.version>3.0.3</google-cloud-dataproc.version>
```
[the remark this answers is not in the document]
Comment (davidrabinowitz) [reply spark-bigquery-parent/pom.xml]: Moved to PR #587

**Review by VinayLondhe14 — APPROVED** [review]

**Pull request merged**